modules/special-rules/pages/cavalry.adoc

```diff
@@ -19,7 +22,6 @@ include::main-rules:partial$charge-moves-split-speed.adoc[]
 ]
 * Make its second Speed value three times (3x) the *first* Speed value of the base unit.
 footnote:split-speed[]
-* Add +1 to its Armour value.
```

Comment:
FYI: I seem to have made a slight mistake in the process of merging the 'footnote' branch into master and then into this PR -- as a result, I unwittingly added this line back in and removed it again manually, hence why it appears here like this... 🤣

Merge conflicts can be a little confusing, and then there are the PR conversations and unapplied suggestions that can get thrown off track too. Hopefully I didn't make any other similar mistakes that I didn't catch, but I'll keep my eyes open just in case.